packages/node_modules/@ciscospark/common/src/uuid-utils.js

```diff
@@ -27,3 +31,29 @@ export function deconstructHydraId(id) {
     type: payload.pop()
   };
 }
+
+/**
+ * Returns file URLs for the activity, adhering to Hydra details,
+ * e.g., https://api.ciscospark.com/v1/contents/Y2lzY29zcGF...
+ * @see https://developer.webex.com/docs/api/v1/messages/get-message-details
+ * @param {Object} activity from mercury
+ * @returns {Array} file URLs
+ */
+export function getHydraFiles(activity) {
+  const hydraFiles = [];
+  const {files} = activity.object;
+
+  if (files) {
+    const {items} = files;
+
+    // Note: Generated ID is dependent on file order.
+    for (let i = 0; i < items.length; i += 1) {
```

Comment:
Could this be a `.forEach` or `.map` on the items array?

Reply:
unless this is dependant on being as fast as possible

Reply:
Unfortunately, no, it's dependent on position. Any other way has unused vars.

Reply:
This is me being extra for the sake of being extra, but couldn't you do something like this?
```javascript
return activity.object.files
  ? activity.object.files.items.map(item =>
      hydraFiles.push(
        `${hydraBaseUrl}/contents/${constructHydraId(hydraTypes.CONTENT, `${activity.id}/${i}`)}`
      )
    )
  : [];
```